### PR TITLE
Update Stream Processor puppet resources for continuous update delivery

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: modules/sp_dashboard/manifests/init.pp [hunk 1]
```diff
@@ -46,47 +33,104 @@
...
+    user        => $user,
+    cwd         => "${distribution_path}",
+    onlyif      => "/usr/bin/test ! -d ${install_path}",
+    subscribe   => File["binary"],
+    refreshonly => true,
+    require     => Package['unzip'],
   }
 
-  # Copy dashboard.sh to installed directory
+  # Copy wso2server.sh to installed directory
```
Comment (VimukthiPerera) [inline modules/sp_dashboard/manifests/init.pp]: Shouldn't this be dashboard.sh?

File: modules/sp_dashboard_master/manifests/startserver.pp [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ----------------------------------------------------------------------------
+
+# Class sp_dashboard::startserver
+#
+class sp_dashboard_master::startserver inherits sp_dashboard_master::params {
+
+  # This class should be used to control Stream processor if required.
```
Comment (VimukthiPerera) [inline modules/sp_dashboard_master/manifests/startserver.pp]: Should this be Stream Processor Dashboard?

File: modules/sp_editor/manifests/init.pp [hunk 3]
```diff
@@ -46,47 +33,104 @@
...
+    user        => $user,
+    cwd         => "${distribution_path}",
+    onlyif      => "/usr/bin/test ! -d ${install_path}",
+    subscribe   => File["binary"],
+    refreshonly => true,
+    require     => Package['unzip'],
   }
 
-  # Copy dashboard.sh to installed directory
+  # Copy wso2server.sh to installed directory
```
Comment (VimukthiPerera) [inline modules/sp_editor/manifests/init.pp]: This should be worker.sh

File: modules/sp_dashboard/manifests/init.pp (hunk 1 above)
Comment (Savidude) [reply modules/sp_dashboard/manifests/init.pp]: Fixed https://github.com/wso2/puppet-sp/pull/9/commits/e7029a60e78fa3bc5ece64afaa86ddffd81b11d4

File: modules/sp_editor/manifests/init.pp (hunk 3 above)
Comment (Savidude) [reply modules/sp_editor/manifests/init.pp]: Shouldn't it be editor.sh?

Comment (ctienshi) [reply modules/sp_editor/manifests/init.pp]: +1

**Pull request merged**